desktop/src/main/java/bisq/desktop/components/paymentmethods/SwishForm.java

```diff
@@ -35,20 +35,31 @@
 import javafx.scene.control.TextField;
 import javafx.scene.layout.GridPane;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import lombok.extern.slf4j.Slf4j;
 
 import static bisq.desktop.util.FormBuilder.addCompactTopLabelTextField;
 import static bisq.desktop.util.FormBuilder.addCompactTopLabelTextFieldWithCopyIcon;
 import static bisq.desktop.util.FormBuilder.addTopLabelTextField;
 
+@Slf4j
 public class SwishForm extends PaymentMethodForm {
-    private static final Logger log = LoggerFactory.getLogger(SwishForm.class);
-
     private final SwishAccount swishAccount;
     private final SwishValidator swishValidator;
     private InputTextField mobileNrInputTextField;
 
+    public SwishForm(PaymentAccount paymentAccount,
+                     AccountAgeWitnessService accountAgeWitnessService,
+                     SwishValidator swishValidator,
+                     InputValidator inputValidator,
+                     GridPane gridPane,
+                     int gridRow,
+                     BSFormatter formatter) {
+        super(paymentAccount, accountAgeWitnessService, inputValidator, gridPane, gridRow, formatter);
+        this.swishAccount = (SwishAccount) paymentAccount;
+        this.swishValidator = swishValidator;
+        this.swishValidator.setIsoCountryCode("SE");
```

Comment:
Maybe that could be done directly in the SwishValidator and not in the Form. What do you think?

Reply:
Oops.  Of course it should be down in the validator.

Reply:
In the SwishValidator constuctor:   super.setIsoCountryCode("SE")   ?

Reply:
As sb just mentioned in Slack #dev channel, probably better this.setIsoCountryCode("SE")